community/modules/scripts/ramble-setup/templates/ramble_setup.yml.tpl

```diff
@@ -36,32 +22,38 @@
     chown_owner: ${chown_owner}
     chgrp_group: ${chgrp_group}
   tasks:
+  - name: Acquire lock
```

Comment:
I believe this task will have to have some `failed_when` behavior defined otherwise failure will kill the setup instead of just skipping the tasks. Something like `failed_when: false`.